<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibFolderForm</class>
 <widget class="QWidget" name="LibFolderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="SelectedFolderName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>FolderName</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUpdate">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>140</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="icon">
    <iconset resource="ServiceLib.qrc">
     <normaloff>:/Qtres/Update.png</normaloff>:/Qtres/Update.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="labelFolder">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Saved folder</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ServiceLib.qrc"/>
 </resources>
 <connections/>
</ui>
